CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,45 @@
+## Expected behavior
+
+ - **Respect**. We may not agree always between us, but this is not a excuse to have bad manners against the other people. I'ts important to mantain a respectull comunity to feel confortable and be more productive.
+
+ - **Be professional**. We are a community of profeswsional, so it's important to remain like that and be careful of the words we choose.
+
+ - **Be responsible**. It is important at work to remain responsible and to act accordingly, since our actions and our contributions can have an effect on other people and projects.
+
+ - **Be polite and kind**. We can find several people working together in this community, and therefore, it is everyone's mission to promote a favorable working and cooperation environment by being kind and polite.
+  
+ - **Be supportive**. As we work, we may find that not all people are at the same level of knowledge as we are. In the case of finding a person with lower levels of knowledge than us, it is important to be understanding and supportive, it is better to help someone learn rather than to be angry and disrespectful. In this way, together we achieve a favorable environment, and in the case of finding ourselves in the opposite situation, we can be sure that we will feel good about receiving help and learning. We all want to grow!
+
+## Unacceptable behavior
+
+Harassment is not tolerated and must be reported. Harassment includes:
+
+- Offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion, sexual images, deliberate intimidation, stalking, sustained disruption, and unwelcome sexual attention.
+
+- Disrespect opinions.
+
+- Intimidation, stalking or bullying.
+
+- Virtual harassment.
+
+Every individual asked to stop any kind of harassment is expected to comply immediately.
+
+We encourage to report any harassment suffered.
+
+## Reporting 
+
+If you are the subject of harassment, notice that someone else is being harassed, or have any other concerns, please contact us by email ____________@wiris.com as soon as possible. All reports will be handled with discretion. In your report please include:
+
+* Your contact information.
+* Names (real, nicknames, or pseudonyms) of any individuals involved. If there are additional witnesses, please include them as well. Your account of what occurred, and if you believe the incident is ongoing. If there is a publicly available record (e.g. a mailing list archive or a public IRC logger), please include a link.
+* Any additional information that may be helpful.
+
+After filing a report, a representative will contact you personally. If the person who is harassing you is part of the response team, they will recuse themselves from handling your incident. A representative will then review the incident, follow up with any additional questions, and make a decision as to how to respond. We will respect confidentiality requests for the purpose of protecting victims of abuse.
```

Comment:
As much as I'd like this to work, we should talk with HR to ensure that we can guarantee this. With these statements we are committing ourselves, we should make sure as a team/company we have the time, resources and formation to deal with this. If you want you can bring this up in the daily meeting so we can plan how to deal with it.